.travis.yml

```diff
@@ -2,11 +2,6 @@ language: objective-c
 
 osx_image: xcode7.2
 
-cache:
```

Comment:
I tested this setting does not do anything, .nvm is empty on every build and npm install is done fresh.
Not too bad, npm is stable recentlly and takes just 3 minutes